code/modules/mob/living/living_movement.dm

```diff
@@ -25,6 +29,10 @@
 	update_move_intent_slowdown()
 	return ..()
 
+/// whether or not we can slide under another living mob. defaults to if we're not dense. CanPass should check "overriding circumstances" like buckled mobs/having PASSMOB flag, etc.
+/mob/living/proc/can_move_under_living(mob/living/other)
+	return !density
```

Comment:
is this supposed to be !other.density ?

Reply:
nope, it just checks if we're not dense, it's a semi-snowflakey-but-not-that-snowflakey way of dealing with carbon crawl-under-ing.

Reply:
then why does it take a mob as an argument thats never used?

Reply:
carbon/can_move_under_living uses it